mdc_analytics/crossmatch/pe.py

```diff
@@ -98,8 +99,10 @@ def process_aframe_pe(
                     f"{events.aframe_graceid.values[idx]}"
                 )
                 results[idx] = None
+    valid_results = [r for r in results if r is not None]
+    all_keys = valid_results[0].keys() if valid_results else []
 
-    for key in PE_KEYS:
```

Comment:
Instead of this, I think something like 

```python
for key in PE_KEYS:
    output = []
    for attr in ["", "_lower", "_upper"]:
         attr = key + attr
         for result in results:
            output.append(result[key] if result is not None else np.nan)
        events["aframe_" + key] = output
        
```
would work

Reply:
@malinadesai I think this is the last comment to address